# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/CookieServiceParent.h
# Commit: 0d57b435d872
# Full Hash: 0d57b435d872751c2a26879fb04bf8745314e37a
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-12 09:23:07
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 8 - CookieStore API - e10s support, r=smaug,cookie-reviewers,timhuang
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D217574
# ==============================================================================

diff -r a63a5bcb0524 -r 0d57b435d872 netwerk/cookie/CookieServiceParent.h
--- a/netwerk/cookie/CookieServiceParent.h	Wed Sep 11 16:26:57 2024 +0000
+++ b/netwerk/cookie/CookieServiceParent.h	Wed Sep 11 16:26:57 2024 +0000
@@ -13,7 +13,11 @@
 class nsICookie;
 namespace mozilla {
 class OriginAttributes;
-}
+
+namespace dom {
+class ContentParent;
+}  // namespace dom
+}  // namespace mozilla
 
 class nsIEffectiveTLDService;
 
@@ -27,7 +31,7 @@
   friend class PCookieServiceParent;
 
  public:
-  CookieServiceParent();
+  explicit CookieServiceParent(dom::ContentParent* aContentParent);
   virtual ~CookieServiceParent() = default;
 
   void TrackCookieLoad(nsIChannel* aChannel);